public/nogaun/index.html

```diff
@@ -15,6 +15,7 @@ <h1>Welcome to NoGaUn page</h1>
         <a class="done" href="#">#0: Registration</a>
         <a class="done" href="issue_1/index.html">#1: Structure, Template, Reset file</a>
         <a class="done" href="issue_2/index.html">#2: Issue 2: Layouts, Master layout</a>
+        <a class="doing" href="issue_3/index.html">#3: Issue 3: Layouts, Components layout</a>
```

Comment:
look good
Please change this line
```suggestion
        <a class="done" href="issue_3/index.html">#3: Issue 3: Layouts, Components layout</a>
```